code/client/munkilib/updatecheck.py

```diff
@@ -2553,6 +2553,42 @@ def getmanifest(partialurl, suppress_errors=False):
       string local path to the downloaded manifest.
     """
     #global MANIFESTS
+
+    def walkManifestCache(path):
```

Comment:
Not really sure why we've added these functions. Can't we just continue using the MANIFESTS dictionary, but use the full (or relative) path as the key instead of just the filename?